hrrecruitment/tests/phpunit/CRM/HRRecruitment/BAO/HRVacancyTest.php

```diff
@@ -49,8 +37,11 @@ protected static function _populateDB($perClass = FALSE, &$object = NULL) {
    */
   function testCreateGet() {
 
-    $permissionIndividual1Id = Contact::createIndividual();
-    $permissionIndividual2Id = Contact::createIndividual();
+    $params1 = array('first_name' => 'micky', 'last_name' => 'mouse');
+    $permissionIndividual1Id = $this->createContact($params1);
```

Comment:
I think it would be better to keep the Contact::createIndividual() call here. Here are the reasons:
- This createContact() method will end up duplicated in another extension tests
- The Contact::createIndividual() is a little bit faster than createContact(), because it doesn't use the API